instance/models/openedx_appserver.py

```diff
@@ -35,22 +35,33 @@
 from instance.models.appserver import AppServer
 from instance.models.mixins.ansible import AnsibleAppServerMixin, Playbook
 from instance.models.mixins.utilities import EmailMixin
-from instance.models.utils import format_help_text
+from instance.models.utils import default_setting, format_help_text
 from instance.openstack_utils import get_openstack_connection, sync_security_group_rules, SecurityGroupRuleDefinition
 from pr_watch.github import get_username_list_from_team
 
 # Constants ###################################################################
 
-DEFAULT_EDX_PLATFORM_REPO_URL = 'https://github.com/{}.git'.format(settings.DEFAULT_FORK)
 
 # OpenStack firewall rules (security group rules) to apply to the main security group of each AppServer:
 OPENEDX_APPSERVER_SECURITY_GROUP_RULES = [
     # Convert this setting from a list of dicts to a list of SecurityGroupRuleDefinition tuples.
     SecurityGroupRuleDefinition(**rule) for rule in settings.OPENEDX_APPSERVER_SECURITY_GROUP_RULES
 ]
 
-# Models ######################################################################
 
+# Functions ###################################################################
+
+def default_admin_organizations():
+    """
+    Helper function to set the default value for the field `github_admin_organizations`.
+    """
+    default = settings.DEFAULT_ADMIN_ORGANIZATION
```

Comment:
@itsjeyd We set default values for all settings in `opencraft/settings.py` (and even if we didn't, adding and removing settings from the environment can't really affect whether a Django setting exists).